include/oboe/Definitions.h

```diff
@@ -1077,6 +1077,24 @@ namespace oboe {
         Always
     };
 
+    /**
+     * The values are defined to be used for the accuracy requirement when calling
```

Comment:
Perhaps we can add a diagram in the Wiki explaining how to use this API. It might be hard for callers to visualize otherwise